src/include/stir/ProjDataInMemory.h

```diff
@@ -92,20 +92,45 @@ class ProjDataInMemory : public ProjDataFromStream
   virtual void axpby(const float a, const ProjData& x,
                      const float b, const ProjData& y);
 
-    
+  //! typedefs to simplify iterator
+  typedef Array<1,float>::full_iterator full_iterator;
+  typedef Array<1,float>::const_full_iterator const_full_iterator;
```

Comment:
doxygen comment only applies for one line, unless you use groups. See e.g. https://github.com/UCL/STIR/blob/fd32b1f516cd7739c4cf292ddfe16144356643e7/src/include/stir/Array.h#L102-L107

Use the opportunity to include typedefs for `iterator` and `const_iterator`.